middleware/route.test

```diff
@@ -125,7 +125,7 @@ test.only ('', async t => {
   , server =
       (new Server (stack)).serve ``
 
-  , response = await fetch (`${URI}${encoded}`)
+  , response = await fetch (`${URI}${value}`)
```

Comment:
Why code more-generic? The original name is so much more clear...

Reply:
@tmornini @brandondees  wrapping this one up just making sure I covered everything asked for. `encoded` in retrospect is better off. "token/value" pairs are the most abstract. The automobile to the skateboard.

#FunFact @tmornini they don't call it HTTP method in the spec they call it `method *token*` ;-) 

![capture d ecran 2018-05-09 a 11 37 57](https://user-images.githubusercontent.com/38223/39825123-0bbe1a0a-537f-11e8-91ca-beacde50b05e.png)